packages/odyssey-react/src/components/Status/Status.module.scss

```diff
@@ -11,50 +11,56 @@
  */
 
 $status-indicator-size: 1em * 0.5;
-$status-border-radius: 1em;
 
 .label {
   display: block;
-  margin-block-end: var(--SpaceRemXs);
-  font-weight: var(--FontWeightBold);
+  margin-block-end: var(--LabelMarginBlockEnd);
+  font-weight: var(--LabelFontWeight);
 }
 
-.value {
+.descriptor {
   position: relative;
-  padding-inline-start: calc(#{$status-indicator-size} + var(--SpaceEmXs));
+  padding-inline-start: calc(
+    #{$status-indicator-size} + var(--DescriptorPaddingInlineStart)
+  );
   background-color: transparent;
 
   &::before {
-    @include border-radius($status-border-radius);
+    @include border-radius(50%);
```

Comment:
Using a `%` rather than `em` results in a different ellipse calculation.

`%` values refer to X/Y individually. Here, that's `50%` of the height and `50%` of the width. What we want is `1em` on both X and Y axes of the border radius, so that the shape is a circle rather than oblong.

[This post](https://stackoverflow.com/questions/29966499/border-radius-in-percentage-and-pixels-px-or-em) has a good visualization.